### ISSUE TITLE
Cannot set machine from deb_arch 'armv7l'

### ISSUE THREAD

**Issue by JimVanEeden** [electron-userland/electron-builder#3701]
* **Version**: 20.38.5

* **Target**:  linux armv7l

I keep getting the error `Cannot set machine from deb_arch 'armv7l'` on my macOS 10.14.3, but actually the same error occurs on an Ubuntu 18.04 machine.

```
  • electron-builder version=20.38.5
  • loaded configuration file=package.json ("build" field)
  • writing effective config file=release/builder-effective-config.yaml
  • no native production dependencies
  • packaging       platform=linux arch=armv7l electron=3.0.10 appOutDir=release/linux-armv7l-unpacked
  • building        target=snap arch=armv7l file=release/fridge-consumer-electron_0.17.1_armv7l.snap
  • building        target=AppImage arch=armv7l file=release/ElectronReact 0.17.1 armv7l.AppImage
Cannot set machine from deb_arch 'armv7l'
  ⨯ exit status 2
github.com/develar/app-builder/pkg/util.ExecuteWithInheritedStdOutAndStdErr
        /Volumes/data/Documents/app-builder/pkg/util/util.go:59
github.com/develar/app-builder/pkg/package-format/snap.buildUsingDocker
        /Volumes/data/Documents/app-builder/pkg/package-format/snap/snap.go:371
github.com/develar/app-builder/pkg/package-format/snap.Snap
        /Volumes/data/Documents/app-builder/pkg/package-format/snap/snap.go:230
github.com/develar/app-builder/pkg/package-format/snap.ConfigureCommand.func1
        /Volumes/data/Documents/app-builder/pkg/package-format/snap/snap.go:103
github.com/alecthomas/kingpin.(*actionMixin).applyActions
        /Volumes/data/go/pkg/mod/github.com/alecthomas/kingpin@v2.2.6+incompatible/actions.go:28
github.com/alecthomas/kingpin.(*Application).applyActions
        /Volumes/data/go/pkg/mod/github.com/alecthomas/kingpin@v2.2.6+incompatible/app.go:557
github.com/alecthomas/kingpin.(*Application).execute
        /Volumes/data/go/pkg/mod/github.com/alecthomas/kingpin@v2.2.6+incompatible/app.go:390
github.com/alecthomas/kingpin.(*Application).Parse
        /Volumes/data/go/pkg/mod/github.com/alecthomas/kingpin@v2.2.6+incompatible/app.go:222
main.main
        /Volumes/data/Documents/app-builder/main.go:78
runtime.main
        /usr/local/Cellar/go/1.11.4/libexec/src/runtime/proc.go:201
runtime.goexit
        /usr/local/Cellar/go/1.11.4/libexec/src/runtime/asm_amd64.s:133
```

**Comment by AlexanderKosianchuk** [comment]
Got the same

**Comment by njho** [comment]
Same. What does this actually mean?

**Comment by andreolaya** [comment]
Got the same, I'm going to try tonight to compile an arm7l machine in this case in a raspberry pi 3 b + and let's hope it happens.

**Comment by Hadatko** [comment]
Same :/

**Comment by Hadatko** [comment]
Hi,
i am not sure if it is correct but
    "linux": {
      "category": "gui",
      "target": [
        "zip"
      ]
in package.json fixed my issue

**Comment by develar** [comment]
Fixed in 20.434.0.
